University-of-Edinburgh-01.svg

```diff
@@ -0,0 +1,1367 @@
+<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Please move this file to the `nf-co.re/public_html/assets/img/contributors-colour/` folder 👍